framework/bootstrap/config.go

```diff
@@ -203,39 +203,22 @@ func (env *Environment) IstioRev() string {
 }
 
 // RevInScope check revision
-// when StrictRev is true, return true if revision in global.IstioRev or global.configRev
-// when StrictRev is false, return true if revision in (global.IstioRev or global.configRev) or revision is empty or global.IstioRev is empty
+// when StrictRev is true, return true if revision in global.IstioRev
+// when StrictRev is false, return true if revision in global.IstioRev or revision is empty or global.IstioRev is empty
 func (env *Environment) RevInScope(rev string) bool {
 
 	if env == nil || env.Config == nil || env.Config.Global == nil {
 		return true
 	}
 
-	configRevs := initConfigRevision(env.Config.Global.IstioRev, env.Config.Global.ConfigRev)
-	revs := strings.Split(configRevs, ",")
-
+	istioRevs := strings.Split(env.Config.Global.IstioRev, ",")
 	if env.Config.Global.StrictRev {
-		return inRevs(rev, revs)
+		return inRevs(rev, istioRevs)
```

Comment:
when `StrictRev == true && IstioRev == ""`, we'll only get  configs witch **eact empty rev** which is not what we expect.

Reply:
when StrictRev == true && IstioRev == ""

slime will manage all resource